src/webview/MessageList.js

```diff
@@ -278,9 +280,9 @@ class MessageList extends Component<Props> {
             // Android. On iOS, though, it involves some unpredictable UUIDs which
             // RN provides no good interface to. (`react-native-fs` is awful.)
             if (
-              url.startsWith('file://')
-              && url.endsWith(baseUrlTail)
-              && !unsafeUrlRegex.test(url)
+              url.startsWith('file://') &&
```

Comment:
As @chrisbobbe suggested [here](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/SOLVED.3A.20Prettier.20auto-formatting/near/893193), I downgraded Prettier VS Code extension to v4.7, but this formatting change still happens. :confused:

Reply:
That would be because of this line :slightly_smiling_face: :
```diff
--- a/src/webview/MessageList.js
+++ b/src/webview/MessageList.js
@@ -1,3 +1,4 @@
+/* eslint-disable operator-linebreak */
 /* @flow strict-local */
```

Effectively that line requests this formatting change, by disabling the way we control that aspect of formatting.